### DISCUSSION TITLE
StreamPark 2.2.0版本登陆后网页是空白的

### DISCUSSION THREAD

**Discussion by wsing007** [apache/streampark#2790] [Q&A]
StreamPark 2.2.0版本登陆后网页是空白的，请哪位前辈帮助一下？

**Comment by onlyotdn** [comment]
metoo

**Comment by onlyotdn** [comment]
> StreamPark 2.2.0版本登陆后网页是空白的，请哪位前辈帮助一下？

My front-end knowledge is lacking, and I don't know the reason for the time being. However, relying on a little front-end knowledge I learned at school, I first started the startup.sh script locally, and then started the front-end service under the webapp, which can be accessed through localhost:10001.
